stdlib/Libdl/docs/src/index.md

```diff
@@ -1,5 +1,9 @@
 # Dynamic Linker
 
+The `Libdl` module in Julia facilitates dynamic linking of shared libraries. This allows Julia programs to
```

Comment:
Maybe you accidentally mixed commits in other PR.

Reply:
Oh sorry about that, thanks for noticing!